assets/__tests__/live_socket_test.js

```diff
@@ -1,32 +1,30 @@
-import '@babel/polyfill';
 import LiveSocket, { View } from '../js/phoenix_live_view';
 import {Socket} from "phoenix"
-import jsdom from 'jsdom';
-const { JSDOM } = jsdom;
+import { JSDOM } from 'jsdom'
 
 function liveViewDOM() {
   const div = document.createElement('div')
-  div.setAttribute('data-phx-view', '');
-  div.setAttribute('data-phx-session', 'abc123');
-  div.setAttribute('id', 'container');
+  div.setAttribute('data-phx-view', '')
+  div.setAttribute('data-phx-session', 'abc123')
+  div.setAttribute('id', 'container')
   div.innerHTML = `
     <label for="plus">Plus</label>
     <input id="plus" value="1" />
     <button phx-click="inc_temperature">Inc Temperature</button>
-  `;
+  `
   const button = div.querySelector('button')
   const input = div.querySelector('input')
   button.addEventListener('click', () => {
     setTimeout(() => {
-      input.value += 1;
-    }, 200);
-  });
+      input.value += 1
+    }, 200)
+  })
```

Comment:
possible to just back out all changes from these two existing files (view / live_socket)?

Reply:
Including the removal of `@babel/polyfill`? Cause that's been removed from `package.json`.